ramls/record-bulk.raml

```diff
@@ -41,3 +41,9 @@ resourceTypes:
           enum: [id]
           default: id
           required: false
+        recordType:
+          description: Determines which record type UUIDs should be fetched
+          type: string
+          enum: [INSTANCE, HOLDINGS]
```

Comment:
A minor one but we have singular for instance but plural for holdings, would be easier to make it consistent

Reply:
for some reason everywhere in folio holdings are used instead of holding, but ok, I will change to holding

Reply:
@KVupp  done